sql/core/src/test/scala/org/apache/spark/sql/collation/DefaultCollationTestSuite.scala

```diff
@@ -475,6 +475,45 @@ class DefaultCollationTestSuiteV1 extends DefaultCollationTestSuite {
       checkAnswer(sql(s"SELECT COUNT(*) FROM $testView WHERE c2 = 'b'"), Seq(Row(0)))
     }
   }
+
+  test("ALTER VIEW check default collation") {
+    Seq("", "TEMPORARY").foreach {
+      temporary =>
```

Comment:
nit: temporary should be in the same line, see examples in https://github.com/databricks/scala-style-guide
```suggestion
    Seq("", "TEMPORARY").foreach { temporary =>
```